elytron/src/main/java/org/wildfly/extension/elytron/ElytronDefinition.java

```diff
@@ -108,6 +108,13 @@
  */
 class ElytronDefinition extends SimpleResourceDefinition {
 
+    /**
+     * System property which if set to {@code true} will cause the JVM wide default {@link SSLContext} to be restored when the subsystem shuts down.
+     *
+     * This property is only for use by test cases.
+     */
+    static final String RESTORE_DEFAULT_SSL_CONTEXT = ElytronDefinition.class.getPackage().getName() + ".restore-default-ssl-context";
```

Comment:
More of a personal preference than anything, but I'd rather just see the whole string hard-coded. If for whatever reason this class gets refactored it may unknowingly break things.